### PR TITLE
Saturn Release Informational Maintenance + RPIP-49

### PR DESCRIPTION
**Pull request by LongForWisdom**
I simplified the entries under 55 and 56 a bit, I think it's better that those sections are primarily just lists of RPIPs rather than trying to explain what exactly is in them, that should probably be left to the RPIPs themselves or a link to a blog post or other more readable document. 

Reordered RPIP-49 list to be in RPIP order as it was bugging me. Minor rewords that I probably didn't need to make.

### CODE REVIEW

**Comment by orangesamus** [comment]
While fixing the order of things do you want to also place the [contents section](https://github.com/rocket-pool/RPIPs/pull/273/files#diff-86facc50e50dfcc098582bab05d3ab5cc019b1adf699e811eed664dc42bd533eL30) in rpip order?

File: RPIPs/RPIP-49.md [hunk 1]
```diff
@@ -100,32 +100,27 @@ This RPIP introduces a delay after protocol upgrades have been confirmed but pri
...
-  * Framework
-  * 4ETH minimum bond
-* [RPIP-59: Deposit Mechanics](RPIP-59.md)
-  * Standard and express queues.
-* [RPIP-46: Universal Adjustable Revenue Split](RPIP-46.md)
-  * Splits between rETH, node operators, and vote-eligible staked RPL staked in megapools
-  * RPL issuance rewards no longer have a minimum stake required
-* RPL Value Capture - Increased share to voting Node Operators.
+* [RPIP-42: Bond curves](RPIP-42.md) - Partial Inclusion
+  * 4 ETH minimum bond
```
Comment (orangesamus) [inline RPIPs/RPIP-49.md:104]:
```suggestion
  * 4 ETH bonds only
```

Comment (orangesamus) [reply RPIPs/RPIP-49.md:104]: Or "4 ETH bonds for all validators"?

File: RPIPs/RPIP-49.md [hunk 2]
```diff
@@ -100,32 +100,27 @@ This RPIP introduces a delay after protocol upgrades have been confirmed but pri
...
+* [RPIP-43: Megapools](RPIP-43.md) (includes ETH-only validators)
```
Comment (orangesamus) [inline RPIPs/RPIP-49.md:105]: I kinda liked the bullet point better :sweat_smile:, but nbd either way. I like "Includes" better than "including" though

File: RPIPs/RPIP-49.md [hunk 3]
```diff
@@ -100,32 +100,27 @@ This RPIP introduces a delay after protocol upgrades have been confirmed but pri
...
+* [RPIP-46: Universal Adjustable Revenue Split](RPIP-46.md) - Partial Inclusion
+  * UARS introduced with no_share, voter_share, and reth_commission
+  * RPL issuance extended to legacy nodes below cliff
```
Comment (orangesamus) [inline RPIPs/RPIP-49.md:108]:
```suggestion
  * RPL issuance rewards extended to all staked RPL
```

**Comment by orangesamus** [comment]
Not important but if you want to go through and keep convention discussed earlier of no periods on bullets (unless multiple sentences) then you could update that... But I think it's fine to leave alone if you don't want to

File: RPIPs/RPIP-49.md (hunk 1 above)
Comment (LongForWisdom) [reply RPIPs/RPIP-49.md:104]: Technically it's 4-8-4-*? So not sure I can say that.

File: RPIPs/RPIP-49.md (hunk 3 above)
Comment (LongForWisdom) [reply RPIPs/RPIP-49.md:108]: Cliff is important to mention I think?

File: RPIPs/RPIP-49.md (hunk 2 above)
Comment (LongForWisdom) [reply RPIPs/RPIP-49.md:105]: Eh, I think I will leave it.

File: RPIPs/RPIP-49.md (hunk 3 above)
Comment (orangesamus) [reply RPIPs/RPIP-49.md]: I think "legacy nodes below cliff" might be confusing (eg, does it include megapool staked RPL?)

Maybe you could put "no more cliff"...
RPL issuance rewards extended to all staked RPL (no more minimum requirement cliff)

File: RPIPs/RPIP-49.md (hunk 1 above)
Comment (Valdorff) [reply RPIPs/RPIP-49.md:104]: I like "4 ETH bond per validator"

File: RPIPs/RPIP-49.md (hunk 3 above)
Comment (Valdorff) [reply RPIPs/RPIP-49.md]: "RPL issuance rewards extended to all staked RPL (no minimum requirement)"?

**Comment by LongForWisdom** [comment]
> While fixing the order of things do you want to also place the [contents section](https://github.com/rocket-pool/RPIPs/pull/273/files#diff-86facc50e50dfcc098582bab05d3ab5cc019b1adf699e811eed664dc42bd533eL30) in rpip order?

Yeah, I'll do this tomorrow / later. Ran out of time today.

File: RPIPs/RPIP-49.md (hunk 1 above)
Comment (LongForWisdom) [reply RPIPs/RPIP-49.md:104]: > What do you mean? Each new validator will require an additional 4 ETH bond. The array will go 4-8-12-16-*, but each index is how much capital you need for the next validator (4th validator requires 16 ETH or 4ETH bond per validator)?
> 
> Maybe "4 ETH bond per validator"?

Possible I've been misunderstanding this for ages lol. Is it not:
4 ETH for first.
12 ETH for second.
16 ETH for third.
?

I thought the array value was the delta that you needed, not the total?

Comment (LongForWisdom) [reply RPIPs/RPIP-49.md:104]: Will have to think it out tomorrow.

**Comment by LongForWisdom** [comment]
So, I looked up bullet formatting a bit. Some rules I found are:
1. Use period if it contains a sentence (even if just one)
2. Don't use a period if it's a single word or item. 
3. Don't use a period if it's completing a 'stem.'
4. Don't mix lists of items and sentences. 

I tweaked some so it was less obviously inconsistent, but bleh, is fine. It's a super minor thing.

File: RPIPs/RPIP-49.md (hunk 1 above)
Comment (orangesamus) [reply RPIPs/RPIP-49.md]: It is total amount you need, so Saturn 1 is flat 4ETH bonds
Total per index: [4,8,12,…]
Alternatively just the “extra cost per index: [4,4,4,…]

Saturn changes to:
[4,8,9.5,11,13.5…]
or
(Additional cost): [4,4,1.5,1.5,1.5,…]

**Comment by orangesamus** [comment]
👍

File: RPIPs/RPIP-49.md [hunk 4]
```diff
@@ -91,41 +82,46 @@ Here we describe the mechanics of Node Operator deposits and validator creation,
...
-  * Including ETH-only validators.
-* [RPIP-42: Bond curves](RPIP-42.md)
-  * Framework
-  * 4ETH minimum bond
-* [RPIP-59: Deposit Mechanics](RPIP-59.md)
-  * Standard and express queues.
-* [RPIP-46: Universal Adjustable Revenue Split](RPIP-46.md)
-  * Splits between rETH, node operators, and vote-eligible staked RPL staked in megapools
-  * RPL issuance rewards no longer have a minimum stake required
-* RPL Value Capture - Increased share to voting Node Operators.
```
Comment (Valdorff) [inline RPIPs/RPIP-49.md:114]: I think the RPL value capture bullet should remain. Can be "RPL Value Capture - Increased share to vote-eligible RPL staked in megapools" to parallel Saturn 2 clearly.

File: RPIPs/RPIP-49.md (hunk 1 above)
Comment (Valdorff) [reply RPIPs/RPIP-49.md]: Confirming samus has it right

**Review by orangesamus — APPROVED** [review]
lgtm now

**Pull request merged**